README.md

```diff
@@ -14,11 +14,10 @@
 
 [![Autotest Status](https://autotest.ardupilot.org/autotest-badge.svg)](https://autotest.ardupilot.org/)
 
-Ardupilot is the most advanced, full-featured and reliable open source autopilot software available. It has
-been under development since 2010 by a diverse team of professional engineers, computer scientists and community contributors.
-Our autopilot software is capable of controlling almost any vehicle system imaginable, from conventional
-airplanes, quadplanes, multirotors, and helicopters, to rovers, boats, balancebots and even submarines. It is continually being expanded to provide
-support for new emerging vehicle types.
+Ardupilot is the most advanced, full-featured and reliable open source autopilot software available.
```

Comment:
```suggestion
ArduPilot is the most advanced, full-featured and reliable open source autopilot software available.
```

Reply:
@peterbarker feel free to fixup and merge